### PR TITLE
[Tests-Only] Split linting, unit, api and acceptance tests

### PR DESCRIPTION
**Pull request by micbar**
## Description

- Try to split the testing pipelines to run them in parallel

## Reason

- CI runs were taking over 1hr

### CODE REVIEW

**Comment by micbar** [comment]
@xoxys could that be that we don't have arm64 and arm architecture on drone.owncloud.com?

**Comment by xoxys** [comment]
Not sure if it works or not, afaik there are scaleway setups to spin up ARM machines. But dont know it was ever user or tested.

**Comment by micbar** [comment]
I made the code prettier. Please review again.

File: .drone.star [hunk 1]
```diff
@@ -196,12 +174,44 @@ def testing(ctx, coreBranch = 'master', coreCommit = '', phoenixBranch = 'master
...
+      generate() +
+      build() +
+      ocisServer() + [
       {
         'name': 'phoenixWebUIAcceptanceTests',
         'image': 'owncloudci/nodejs:11',
         'pull': 'always',
         'environment': {
-          'SERVER_HOST': 'http://ocis-server:9100',
+          'SERVER_HOST': 'http://    ocis-server:9100',
```
Comment (individual-it) [inline .drone.star:214]: random spaces?

Comment (PVince81) [inline .drone.star:214]: weird spacing??

File: .drone.star [hunk 2]
```diff
@@ -216,9 +226,9 @@ def testing(ctx, coreBranch = 'master', coreCommit = '', phoenixBranch = 'master
           'git clone -b %s --single-branch --no-tags https://github.com/owncloud/phoenix.git /srv/app/phoenix' % (phoenixBranch),
           'cp -r /srv/app/phoenix/tests/acceptance/filesForUpload/* /uploads',
           'cd /srv/app/phoenix',
-		] + ([
+        ] + ([
           'git checkout %s' % (phoenixCommit)
-		] if phoenixCommit != '' else []) + [
+        ] if phoenixCommit != '' else []) + [
```
Comment (individual-it) [inline .drone.star:231]: indent look strange

File: .drone.star (hunk 1 above)
Comment (micbar) [reply .drone.star:214]: corrected.

File: .drone.star (hunk 2 above)
Comment (micbar) [reply .drone.star:231]: you were using tabs,
i am using spaces now like in the restt of the file?

File: .drone.star [hunk 3]
```diff
@@ -372,7 +329,7 @@ def docker(ctx, arch):
       },
     ],
     'depends_on': [
-      'testing',
+      'acceptanceTests',
```
Comment (phil-davis) [inline .drone.star:332]:
```suggestion
      'linting',
      'unitTests',
      'apiTests',
      'acceptanceTests',
```

File: .drone.star [hunk 4]
```diff
@@ -529,7 +460,7 @@ def binary(ctx, name):
...
```
Comment (phil-davis) [inline .drone.star:463]:
```suggestion
      'linting',
      'unitTests',
      'apiTests',
      'acceptanceTests',
```

Comment (micbar) [reply .drone.star:482]: done

File: .drone.star (hunk 3 above)
Comment (micbar) [reply .drone.star:335]: done

**Comment by micbar** [comment]
merged, but wrong in the UI

**Pull request closed without merging**